WooCommerce/src/main/kotlin/com/woocommerce/android/ui/orders/creation/giftcards/EditGiftCardScreen.kt

```diff
@@ -32,25 +33,33 @@ fun EditGiftCardScreen(viewModel: OrderCreateEditGiftCardViewModel) {
 @Composable
 fun EditGiftCardScreen(
     giftCardValue: String,
+    isValidCode: Boolean,
     onTextChanged: (String) -> Unit,
     onDoneClicked: () -> Unit
 ) {
     Column(
         modifier = Modifier.fillMaxSize()
             .padding(dimensionResource(id = R.dimen.major_100))
     ) {
+        val textFieldLabel = if (isValidCode) {
+            stringResource(id = R.string.order_creation_gift_card_text_field_hint)
+        } else {
+            stringResource(id = R.string.order_creation_gift_card_text_error)
+        }
         WCOutlinedTextField(
             value = giftCardValue,
```

Comment:
np: WDYT if we made the gift card text upper case?

```suggestion
            value = giftCardValue.toUpperCase(Locale.current),
```